backend/database/database.go

```diff
@@ -391,6 +394,43 @@ func (d *Database) ReviewsByVendorID(vendorID uuid.UUID) ([]Review, error) {
 	return result, rows.Err()
 }
 
+//query to find out if the vendor has been starred, it returns true if it is false if not
+func (d *Database) StarredVendorFavorite(VendorID uuid.UUID, ReviewsID uuid.UUID) (bool, error) {
+	var starred bool
+
+	if err := d.db.QueryRow("SELECT * FROM Reviews WHERE ID >= ?",
```

Comment:
You probably meant to say `ID = ?`. You're also passing two arguments when there is only one parameter in the query.